modules/creating-a-github-application-in-administrator-perspective.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: PROCEDURE
+[id="creating-a-github-application-in-administrator-perspective_{context}"]
+
+= Creating a GitHub application in administrator perspective
+
+As a cluster administrator, you can configure your GitHub application with the OpenShift Container Platform cluster to use pipeline as code. This allows you to execute a set of tasks required for build deployment.
```

Comment:
`This` is a weak subject. How about, `This configuration allows you to execute...` 

or 

`Using pipeline as code allows you to execute...`

Reply:
done